### DISCUSSION TITLE
Use Sherpa sherpa-ncnn / sherpa-onyx and get n best phone theories like you could with Kaldi?

### DISCUSSION THREAD

**Discussion by johnburkey** [k2-fsa/sherpa#375] [General]
Like per frame- can you do that with k2/sherpa/sherpa-ncnn / sherpa-onyx

**Comment by johnburkey** [comment]
For that matter, n best BPE tokens would work.  Looks like you can do that?   Anyone think any particular one of the sample models is best for that?

**Reply by csukuangfj** [reply]
All transducer models from icefall support returning n best paths, where each path consists of a sequence of tokens, e.g., BPE tokens.
